Tests/TableView/TableViewDriverTests.swift

```diff
@@ -56,77 +56,81 @@ final class TableViewDriverTests: XCTestCase {
-            XCTAssertEqual(self._tableViewDataSource.tableView(self._tableView, numberOfRowsInSection: $0), $1)
+            XCTAssertEqual(self._tableViewDataSource.tableView(testTableView, numberOfRowsInSection: $0), $1)
         }
     }
 
     /// Table view rows described in the table view model are converted into views correctly.
     func testTableViewRows() {
+        let testTableView = self._tableViewDataSource.testTableView
         parameterize(cases: (0, 44), (1, 42), (2, 42), (9, 44)) {
-            XCTAssertEqual(self._tableViewDataSource.tableView(self._tableView, heightForRowAt: path($0)), $1)
+            XCTAssertEqual(self._tableViewDataSource.tableView(testTableView, heightForRowAt: path($0)), $1)
         }
 
         parameterize(cases: (0, UITableViewCellEditingStyle.none), (1, .delete), (2, .delete), (9, .none)) {
-            XCTAssertEqual(self._tableViewDataSource.tableView(self._tableView, editingStyleForRowAt: path($0)), $1)
+            XCTAssertEqual(self._tableViewDataSource.tableView(testTableView, editingStyleForRowAt: path($0)), $1)
         }
 
         parameterize(cases: (0, true), (1, false), (2, false), (9, true)) {
-            XCTAssertEqual(self._tableViewDataSource.tableView(self._tableView, shouldHighlightRowAt: path($0)), $1)
-            XCTAssertEqual(self._tableViewDataSource.tableView(self._tableView, shouldIndentWhileEditingRowAt: path($0)), $1)
+            XCTAssertEqual(self._tableViewDataSource.tableView(testTableView, shouldHighlightRowAt: path($0)), $1)
+            XCTAssertEqual(self._tableViewDataSource.tableView(testTableView, shouldIndentWhileEditingRowAt: path($0)), $1)
         }
     }
 
     /// Table view section headers described in the table view model are converted into views correctly.
     func testExistingSectionHeaders() {
+        let testTableView = self._tableViewDataSource.testTableView
         let section = 0
         let indexKey = path(section)
-        let header = self._tableViewDataSource._getHeader(section)
+        let header: HeaderView? = self._tableViewDataSource._getHeader(section)
         XCTAssertEqual(header?.label, "title_header+A")
         XCTAssertEqual(header?.accessibilityIdentifier, "access_header+0")
 
-        guard let onScreenHeader = self._tableViewDataSource.tableView(self._tableView, viewForHeaderInSection: indexKey.section) as? TestTableViewSectionHeaderFooter else {
-            XCTFail("Did not find the on screen TestTableViewSectionHeaderFooter header")
+        guard let onScreenHeader = self._tableViewDataSource.tableView(testTableView, viewForHeaderInSection: indexKey.section) as? HeaderView else {
+            XCTFail("Did not find the on screen HeaderView header")
             return
         }
         XCTAssertEqual(onScreenHeader.label, "title_header+A")
-        XCTAssertNil(self._tableView.headerView(forSection: indexKey.section))
+        XCTAssertNotNil(self._tableView.headerView(forSection: indexKey.section))
```

Comment:
this flipped? seems like it's correct now. @jessesquires did I mess this up? or does this seem right?

Reply:
Not sure. I didn't write any of these tests and they're all mostly really bad 😄 😅